taxonomy/models.py

```diff
@@ -0,0 +1,31 @@
+"""Models for lore taxonomy application"""
+
+from django.db import models
+
+
+class Vocabulary(models.Model):
+    repository = models.ForeignKey("Repository")
+    name = models.TextField()
+    description = models.TextField()
+    required = models.BooleanField()
+    type = models.CharField(max_length=1, choices=(
```

Comment:
I would probably just change it to vocabulary_type, it adds some typing and redundancy, but is clear, and underscore naming it would break the private variable/method convention